_common/README-check-version-env-vars.md

```diff
@@ -1,4 +1,4 @@
-## Step 3: [OPTIONAL] Confirm Software Version and Environment Variables
+## [OPTIONAL] Confirm software version and environment variables
```

Comment:
Do we really need a new `_common` directory for a single file?

Reply:
Yes, we have three to four samples using this file at the moment and more to come. If there are other common files in the future, we can add them to this folder.